before-each.js

```diff
@@ -0,0 +1,71 @@
+const fs = require( 'fs' ),
```

Comment:
I guess those are some "before each helpers"? Not a code which is executed as is before each test? Maybe we could simply have some helpers file with all the functions used to run tests (e.g. the ones in `index.js` are also used to setup/teardown/verify tests)?

Reply:
Or you can try to split it contextually - like helpers for file handling, for workflows handling and so on?